dashboards/dev-platform/template1.csv

```diff
@@ -0,0 +1,3 @@
+team,email,samstackname1
```

Comment:
add a new column for grouping - then you can use that for rendering dashboards with groups on if needed.

Reply:
Grouping will be a future continuous improvement. It will require restructuring the data and further csv files. Similar concept used in https://github.com/govuk-one-login/configuration/ repo